src/main/java/org/folio/circulation/rules/Text2Drools.java

```diff
@@ -16,6 +16,8 @@
 import org.antlr.v4.runtime.tree.ParseTreeWalker;
 import org.antlr.v4.runtime.tree.TerminalNode;
 import org.apache.commons.lang3.StringEscapeUtils;
+import org.apache.logging.log4j.util.TriConsumer;
```

Comment:
I'd not use this class, as it is from log4j library, so we will have a dependency on this library.

Reply:
Ok, I've created PolicyValidator interface instead of this one.
Thanks.